win32ss/gdi/ntgdi/path.c

```diff
@@ -1444,7 +1444,7 @@ PATH_FlattenPath(PPATH pPath)
  *
  * Creates a region from the specified path using the specified polygon
  * filling mode. The path is left unchanged. A handle to the region that
```

Comment:
That would be more `Fills Rgn from the specified path etc.` There is no handle allocated here.

Reply:
Updated.